src/printer.js

```diff
@@ -1919,6 +1920,10 @@ function printArgumentsList(path, options, print) {
 
   if (groupLastArg) {
```

Comment:
at the end of the groupLastArg big condition add && (!penultimateArg || penultimateArg.type !== lastArg.type)